climpred/bootstrap.py

```diff
@@ -592,17 +694,48 @@ def bootstrap_compute(
     )
     if 'init' in init_skill:
         init_skill = init_skill.mean('init')
+
     # uninit skill as mean resampled uninit skill
-    uninit_skill = uninit.mean('iteration')
+    uninit_skill = bootstrapped_uninit_skill.mean('iteration')
     if not metric.probabilistic:
         pers_skill = reference_compute(
             hind, verif, metric=metric, alignment=reference_alignment, **metric_kwargs
         )
+        pers_output = True
     else:
+        pers_output = False
         pers_skill = init_skill.isnull()
```

Comment:
This is replicated exactly above I think. Only one of these instances should be here, right?

Reply:
I also don't understand the persistence bootstrapped skill. In this instance it's just doing one persistence computation of `verif`. In the above copied instance it does the same thing but adds a dimension with the same length as `iterations`.

Are we supposed to bootstrap the `verif` in some way to give a distribution? In the case of `resample_dim='init'` do you get a distribution of persistence values since it changes the initializations in which persistence is calculated over? I genuinely don't remember if this is supposed to be a fixed value or not.